superset-frontend/src/dashboard/components/nativeFilters/state.ts

```diff
@@ -42,3 +53,22 @@ export function useFilterSetter(id: string) {
     [id, dispatch],
   );
 }
+export function useFilterScopeTree(): {
+  treeData: [TreeItem];
+  layout: Layout;
+} {
+  const layout = useSelector<RootState, Layout>(
+    ({ dashboardLayout: { present } }) => present,
+  );
+
+  const charts = useSelector<RootState, Charts>(({ charts }) => charts);
+
+  const tree = {
+    children: [],
+    key: DASHBOARD_ROOT_ID,
+    type: DASHBOARD_ROOT_TYPE,
+    title: t('All Panels'),
+  };
+  buildTree(layout[DASHBOARD_ROOT_ID], tree, layout, charts);
```

Comment:
should `useMemo` to only recompute this when necessary

Reply:
👍 will fix in next branch